packages/expo-av/src/Audio/Recording.types.ts

```diff
@@ -7,6 +7,7 @@ export type RecordingStatus = {
   durationMillis: number;
   metering?: number;
   uri?: string | null;
+  mediaServicesReset?: boolean;
```

Comment:
We'll need some documentation around this option 🤔 Additionally I see it's iOS-only? It's should be documented too.